### PR TITLE
add AM_MobileNet1D

### PR DESCRIPTION
**Pull request by WangYizhang01**

### CODE REVIEW

File: AM-MobileNet1D/infer.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+OPTIONS_PATH='cfg/AM_MobileNet1D_TIMIT.cfg'
+PRETRAIN_MODELS='pretrain_models'
+
+if [ ! -d "$PRETRAIN_MODELS" ]; then
+    wget https://oneflow-public.oss-cn-beijing.aliyuncs.com/model_zoo/audio/AM_MobileNet1D.zip
+    unzip AM_MobileNet1D.zip
+    rm -fr AM_MobileNet1D.zip
+fi
+
+python infer.py --cfg $OPTIONS_PATH
```
Comment (CPFLAME) [inline AM-MobileNet1D/infer.sh:13]: python指令替换为python3

File: AM-MobileNet1D/train.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+fi
+
+if [ ! -d "$DATA_PREPARED" ]; then
+    python utils/TIMIT_preparation.py $TIMIT_FOLDER $DATA_PREPARED $LABEL
+fi
+
+echo "Data prepared !"
+echo "Starting training ..."
+
+python train.py --cfg $OPTIONS_PATH
```
Comment (CPFLAME) [inline AM-MobileNet1D/train.sh:30]: python指令替换为python3

**Comment by CPFLAME** [comment]
运行train.sh的时候会报错
```
Traceback (most recent call last):
  File "train.py", line 107, in <module>
    batch_size, data_folder, wav_lst_tr, snt_tr, wlen, lab_dict, 0.2
  File "/home/chengpeng/oneflow_models/models_0.5/AM-MobileNet1D/utils/data_utils.py", line 105, in create_batches_rnd
    [signal, fs] = sf.read(data_folder + wav_lst[snt_id_arr[i]])
  File "/home/dev/.local/lib/python3.6/site-packages/soundfile.py", line 257, in read
    subtype, endian, format, closefd) as f:
  File "/home/dev/.local/lib/python3.6/site-packages/soundfile.py", line 629, in __init__
    self._file = self._open(file, mode_int, closefd)
  File "/home/dev/.local/lib/python3.6/site-packages/soundfile.py", line 1184, in _open
    "Error opening {0!r}: ".format(self.name))
  File "/home/dev/.local/lib/python3.6/site-packages/soundfile.py", line 1357, in _error_check
    raise RuntimeError(prefix + _ffi.string(err_str).decode('utf-8', 'replace'))
RuntimeError: Error opening 'data_preprocessed//train/dr3/mddc0/sx429.wav': System error.
```
但是我在当前目录下找到了`data/lisa/data/timit/raw/TIMIT/TRAIN/DR3/MDDC0/SX429.WAV`，check一下是不是路径写错了？

**Comment by WangYizhang01** [comment]
> 运行train.sh的时候会报错
> 
> ```
> Traceback (most recent call last):
>   File "train.py", line 107, in <module>
>     batch_size, data_folder, wav_lst_tr, snt_tr, wlen, lab_dict, 0.2
>   File "/home/chengpeng/oneflow_models/models_0.5/AM-MobileNet1D/utils/data_utils.py", line 105, in create_batches_rnd
>     [signal, fs] = sf.read(data_folder + wav_lst[snt_id_arr[i]])
>   File "/home/dev/.local/lib/python3.6/site-packages/soundfile.py", line 257, in read
>     subtype, endian, format, closefd) as f:
>   File "/home/dev/.local/lib/python3.6/site-packages/soundfile.py", line 629, in __init__
>     self._file = self._open(file, mode_int, closefd)
>   File "/home/dev/.local/lib/python3.6/site-packages/soundfile.py", line 1184, in _open
>     "Error opening {0!r}: ".format(self.name))
>   File "/home/dev/.local/lib/python3.6/site-packages/soundfile.py", line 1357, in _error_check
>     raise RuntimeError(prefix + _ffi.string(err_str).decode('utf-8', 'replace'))
> RuntimeError: Error opening 'data_preprocessed//train/dr3/mddc0/sx429.wav': System error.
> ```
> 
> 但是我在当前目录下找到了`data/lisa/data/timit/raw/TIMIT/TRAIN/DR3/MDDC0/SX429.WAV`，check一下是不是路径写错了？

我又跑了下没有复现这个问题，这个路径是没问题的。报错里面提示是 data_preprocessed// 双斜杠的问题，可能是系统环境导致的没法复现。我修改了下路径的写法，你再试下有没有这个问题。

File: AM-MobileNet1D/train.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+fi
+
+if [ ! -d "$DATA_LISTS" ]; then
+    wget https://oneflow-public.oss-cn-beijing.aliyuncs.com/datasets/AM_MobileNet1D/data_lists.zip
+    unzip data_lists.zip
+    rm -fr data_lists.zip
+fi
+
+if [ ! -d "$DATA_PREPARED" ]; then
+    python utils/TIMIT_preparation.py $TIMIT_FOLDER $DATA_PREPARED $LABEL
```
Comment (CPFLAME) [inline AM-MobileNet1D/train.sh:24]: 这一个指令我这边没有成功执行，改成python3吧

Comment (CPFLAME) [reply AM-MobileNet1D/train.sh:24]: 应该就是这个问题

**Review by CPFLAME — APPROVED** [review]

**Pull request merged**